src/components/Datathon/Leaderboard/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { connectDB } from '@/db/config';
+import React from 'react';
+import Leaderboard, { ITeamData } from '@/db/models/leaderboard';
+
+export default async function LeaderboardTable() {
+  connectDB();
+
+  const leaderboards = await Leaderboard.find({})
+    .sort({ timestamp: -1 })
+    .limit(1)
+    .exec();
+  const latestLeaderboard = leaderboards[0];
+  latestLeaderboard.data.sort((a: ITeamData, b: ITeamData) => {
+    if (a.score > b.score) {
+      return -1;
+    } else if (a.score < b.score) {
+      return 1;
+    } else {
+      return 0;
+    }
+  });
+
+  return (
+    <section>
+      <div className="container mx-auto my-8 text-white">
+        <h2 className="text-4xl font-bold mb-4 text-center">Leaderboard</h2>
+        <h3 className="text-xl font-bold mb-4">
+          Last Updated: {leaderboards[0].timestamp.toLocaleString()}
+        </h3>
+        <table className="min-w-full border border-white">
+          <thead>
+            <tr>
+              <th className="py-2 px-4 border-b text-center">Position</th>
+              <th className="py-2 px-4 border-b text-center">Team</th>
+              <th className="py-2 px-4 border-b text-center">Score</th>
+              <th className="py-2 px-4 border-b text-center">Attempts Left</th>
```

Comment:
Would like you add some `More Info Icons` here that would give some context for these fields

Score: Aggregated from lastest submissions
Attempts Left: Aggregated over all submissions

Use above as some basic starting text, will get back to you with exact text